test/functional/ws/transport/transactions.js

```diff
@@ -30,63 +30,41 @@ function postTransaction(transaction, cb) {
 		{
 			transactions: [transaction],
 		},
-		cb,
+		() => {},
 		true
 	);
+	cb();
 }
 
 describe('Posting transaction (type 0)', () => {
-	var account;
-	var transaction;
-	var error;
-	var response;
-	var goodTransactions = [];
-	var badTransactions = [];
+	let transaction;
+	const goodTransactions = [];
+	const badTransactions = [];
+	const account = randomUtil.account();
 
 	beforeEach(done => {
-		account = randomUtil.account();
 		transaction = randomUtil.transaction();
 		done();
 	});
 
-	describe('when sender has no funds for a transaction in batch', () => {
-		beforeEach(done => {
+	describe('transaction processing', () => {
+		it('when sender has no funds should fail', done => {
 			transaction = lisk.transaction.createTransaction(
 				'1L',
 				1,
 				account.password
 			);
-			postTransaction(transaction, (err, res) => {
-				error = err;
-				response = res;
+
+			postTransaction(transaction, () => {
 				done();
 			});
 		});
 
-		// For peer-to-peer communiation, the peer does not need to send back
-		// an error message if one of the transactions in the batch fails.
-		// Either the peer acknowledges the receipt of the batch or their don't.
-		it('operation should succeed', () => {
-			badTransactions.push(transaction);
-			expect(error).to.be.null;
-			return expect(response).to.have.property('success').to.be.ok;
-		});
-	});
-
-	describe('when sender has funds for a transaction in batch', () => {
-		beforeEach(done => {
-			postTransaction(transaction, (err, res) => {
-				error = err;
-				response = res;
+		it('when sender has funds should be ok', done => {
+			postTransaction(transaction, () => {
```

Comment:
Transaction is no longer pushed to `goodTransactions`, so what are we testing exactly?